UserApi/UserApi.UnitTests/Services/UserAccountService/IsUserAdminAsyncTests.cs

```diff
@@ -0,0 +1,48 @@
+﻿using System.Collections.Generic;
+using System.Net;
+using System.Threading.Tasks;
+using FluentAssertions;
+using Microsoft.Graph;
+using Moq;
+using NUnit.Framework;
+using Testing.Common.Helpers;
+using UserApi.Services.Models;
+
+namespace UserApi.UnitTests.Services.UserAccountService
+{
+    public class IsUserAdminAsyncTests : UserAccountServiceTests
+    {
+        [TestCase("ADMIN_ROLE_ID", true)]
+        [TestCase("OTHER_ROLE_ID", false)]
+        public async Task Checks_if_user_is_administrator(string userRoleId,  bool expectedResult)
+        {
+            const string principalId = "55c07278-7109-4a46-ae60-4b644bc83a31";
+
+            var endpoint1 = $"{GraphApiSettings.GraphApiBaseUri}beta/rolemanagement/directory/roleAssignments?$filter=principalId eq '{principalId}'";
```

Comment:
looks like a typo here, rolemanagement - if it's supposed to be camel cased like endpoint2. Probably won't make a difference to the graph api call though.